client/electron/vpn_service.ts

```diff
@@ -72,13 +69,10 @@ export async function establishVpn(request: StartRequestJson) {
   };
 
   await invokeGoMethod('EstablishVPN', JSON.stringify(config));
-  statusCb?.(currentRequestId, TunnelStatus.CONNECTED);
 }
 
 export async function closeVpn(): Promise<void> {
-  statusCb?.(currentRequestId!, TunnelStatus.DISCONNECTING);
   await invokeGoMethod('CloseVPN', '');
-  statusCb?.(currentRequestId!, TunnelStatus.DISCONNECTED);
 }
 
 export type VpnStatusCallback = (id: string, status: TunnelStatus) => void;
```

Comment:
Should we allow callbacks to return values?

I could imagine the Go code calling TS code to get values (e.g. localize a string) or to do actions (e.g. show a dialog) that return values.

Reply:
Maybe, we can allow the callback to return a string.